config/initializers/salesforce_services.rb

```diff
@@ -0,0 +1,6 @@
+### shortcut to alias SalesforceServices
+# NOTE: if you update any of these services (new methods, etc),
+#       you will need to restart spring in order for the alias to "refresh"
+AccountService = SalesforceService::AccountService
```

Comment:
It's fine to declare at the top of the controller too, doesn't hurt anything... in fact the benefit there is it _will_ reload the service definition every time you make a request, so while in dev, if you're updating the service you don't have to restart your server to have the alias pick up the new changes. :shrug: